src/main/java/com/williamfiset/algorithms/search/QuickSelect.java

```diff
@@ -0,0 +1,39 @@
+package com.williamfiset.algorithms.search;
+
```

Comment:
Can you add a header describing the algorithm, what it's normally used for, what type of time complexity to expected, etc...

Reply:
Done